presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -417,6 +426,27 @@ private static SliceWriteFunction typedVarcharWriteFunction(String jdbcTypeName)
         };
     }
 
+    private static SliceWriteFunction uuidWriteFunction()
+    {
+        return (statement, index, value) -> {
+            UUID uuid = UUID.nameUUIDFromBytes(value.getBytes());
```

Comment:
This is not correct. It’s creating a UUID using an MD5 of the bytes. Unfortunately, there is no direct conversion from bytes. You need to convert to longs and use the constructor.

The round trip tests should have caught that the value didn’t round trip properly. It’s concerning that they did not.

Reply:
Sorry... the test should have caught that, but I got lazy and missed the forest for the trees.

This part used to fail with a JVM crash, and when I got it fixed I figured that a bout of manual testing should do and then let travis handle the rest.

Evidently that was premature on my part. 😓